### ISSUE TITLE
dpd fails to start on Windows 7

### ISSUE THREAD

**Issue by ctmay4** [deployd/deployd#334]
I've read the other issues about users having trouble getting dpd to start.  However everyone seems to be on OS X or Linux.  I am running Windows 7 and do not have admin privileges.  I have node and mongodb already installed.  I installed dpd using npm:

> npm install deployd -g

Everything looks like it installed correctly:

> C:\Prj\test-dpd>dpd --version
> 0.6.10
> C:\Prj\test-dpd>mongo --version
> MongoDB shell version: 2.4.9

I start up dpd and get this:

> C:\Prj\test-dpd>dpd -d
> starting deployd v0.6.10...
> 
> C:\Prj\test-dpd>

There is no server responding to http://localhost:2403/dashboard.  A data directory is getting created in the project:

> C:\Prj\test-dpd\data>dir
> 02/26/2014  02:58 PM    <DIR>          .
> 02/26/2014  02:58 PM    <DIR>          ..
> 02/26/2014  02:49 PM        16,777,216 local.0
> 02/26/2014  02:49 PM         4,194,304 local.ns
> 02/26/2014  02:58 PM                 5 mongod.lock

I'm not sure where else to look.  I see people posting about starting with "DEBUG", but that method doesn't work on Windows.  I wish there was more feedback from the command-line to indicate the problem.  I'm not sure how to debug this issue.

**Comment by ervo21** [comment]
i'm encountering a compatibility issue in installing deployd. Whenever i'm running 'dpd create sportsstore' it says that the version of my node.exe (in deployd installation folder) is not compatible in the version of my windows. I'm pretty sure that i've downloaded the correct version of node.exe..for deployd, i just selected the windows installer. How could i fix this? :(

**Comment by ericfong** [comment]
Personally I have not tried in windows machine. 

Maybe you can also ask that in https://groups.google.com/forum/#!forum/deployd-users

**Comment by ervo21** [comment]
Hi @ericfong , thank you!

**Comment by Kvarkas** [comment]
or you can ask me? :) I am windows user - could you please specify steps that you are doing?

**Comment by Kvarkas** [comment]
and try do update before start using ;) if got many errors - update json file from github

**Comment by ervo21** [comment]
@Kvarkas thanks in advance. :)
 
In the same location of my nodejs installation folder i'm entered the 'dpd create sportsstore' and it says: 

"C:\Program Files\nodejs\sportsstore>dpd create sportsstore
This version of C:\Program Files\Deployd\tools\node.exe is not compatible with
he version of Windows you're running. Check your computer's system information
o see whether you need a x86 (32-bit) or x64 (64-bit) version of the program, a
d then contact the software publisher."

I'm using win 7, 32-bit. :)

**Comment by Kvarkas** [comment]
I am still frustrating how you get you deployd version - are you install via .exe file from site, or download it from github? Also deployd has his own node.exe witch should be compatible i suppose and not related to you current node.exe version.

**Comment by ervo21** [comment]
I installed deployd via .exe file. Got it from deployd.com...You're right. it has his own node.exe and i'm really confused as to why i'm getting this mind boggling messages.

**Comment by Kvarkas** [comment]
could you please try to do update? its should be shortcut for that created...  and provide version of it (cmd > dpd -V)

**Comment by ervo21** [comment]
this may sound crazy yet whenever i'm typing dpd -V i'm still getting the message:

"C:\Program Files\nodejs\sportsstore>dpd -V
This version of C:\Program Files\Deployd\tools\node.exe is not compatible with
he version of Windows you're running. Check your computer's system information
o see whether you need a x86 (32-bit) or x64 (64-bit) version of the program, a
d then contact the software publisher."

**Comment by thierryotis** [comment]
By uninstalling deployd and by installing by npm install deployd, it worked fine for me. 
This is how i've solved the issue : 
"C:\Program Files\nodejs\sportsstore>dpd -V
This version of C:\Program Files\Deployd\tools\node.exe is not compatible with
he version of Windows you're running. Check your computer's system information
o see whether you need a x86 (32-bit) or x64 (64-bit) version of the program, a
d then contact the software publisher."

**Comment by pkosenko** [comment]
I had trouble getting deployd to work with the Windows installer on Windows 7.  I decided to install it via NPM instead, with node 0.10.35 (haven't yet upgraded to Node 4), and it worked fine, except for the "resources" command (noted elsewhere in bugs  However I tried, .cmd scripts seemed to want to use my existing global Node cache when I tried running them, and, of course, the deployd scripts were not there.  So I gave in and installed the following:

$ npm install -g deployd

That got me out of the woods.  Except that I cannot figure out what port the Mongo database is running on (in order to view it with Robomongo.   I think that reporting the database port (not just the web server port) would be helpful.
